### DISCUSSION TITLE
Style "Fooocus V3"

### DISCUSSION THREAD

**Discussion by AnalyticsXXI** [lllyasviel/Fooocus#3076] [General]
Just sharing some news: there is a project called "Omost" — https://github.com/lllyasviel/Omost which brings the idea of "Fooocus V2" style to the whole new level. It splits canvas into squares and uses LLM to describe what in each square, which allows to create images, where SD follows prompt in terms of composition, different characters, etc. Very interesting thing, something like automated region masks, which just created from the prompt. Maybe something from it can be incorporated into Fooocus.

it also has memory of conversation, so you can discuss with LLM, what you would like to change in previous image and it will try to change it for you. Automated inpainter :)

**Comment by xhoxye** [comment]
It's great, it's translatable, it can prompt extensions

**Comment by PeterJared** [comment]
That looks very good. The hugging page doesn't have available GPUs for rendering when I was trying to use it . Also the local installation instructions are too advanced for my basic understanding of this. I guess this will evolve or even will become part of Fooocus.
